### ISSUE TITLE
Build out the local developers section

### ISSUE THREAD

**Issue by myared** [louisvilleio/louisville.io#24]
As an example: 

http://www.newhaven.io/developers

**Comment by ericlathrop** [comment]
Yeah, interesting. Should louisville.io be the origin of this data, or should it pull it from somewhere else?

**Comment by ericlathrop** [comment]
Right now we're just linking to this kinda inaccurate github search: https://github.com/search?utf8=%E2%9C%93&q=location%3Alouisville

**Comment by myared** [comment]
Ideally, pull. I was thinking they do something similar and inaccurately pull from github, but I haven't looked at it yet. This is issue is more of a reminder for me to work on this at some point.

**Comment by kacole2** [comment]
why not loop through the members of this group and just pull their github profiles?

**Comment by slogsdon** [comment]
If using the GitHub team as a source of truth, `https://api.github.com/orgs/louisvilleio/members` could be used to pull that data. By itself, it will only show public members, but non-public members could be included if that application authenticates against the api as a member of the group

**Comment by sean-clayton** [comment]
https://github.com/search?utf8=%E2%9C%93&q=location%3Alouisville <- Keep in mind this includes people from Louisville, Colorado :wink:
